task-5/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {
+
+    var [fuel, setFeul] = useState(0);
+    var [alertText, setAlertTxt] = useState('Processing...');
+
+    useEffect(() => {
+      fetch('https://new.world.com/fleet/121')
+        .then(response => response.json())
+        .then(json => {
+          setFeul(json);
+          console.log(json);
+        })
+    });
+
+    useEffect(() => {

```

Comment:
This whole thing would be better in `useMemo`, as it's just a changing string based on some inputs.
Even better, you can have this inline without using any react hooks, as using `useMemo` is more performance heavy than just having this all inline, since there's no difficult calculation here.